scheduler/src/main/java/io/camunda/zeebe/scheduler/AsyncClosable.java

```diff
@@ -17,5 +17,5 @@ public interface AsyncClosable {
    *
    * @return the future, which is completed when resources are closed
    */
-  ActorFuture<Void> closeAsync();
+  CompletableActorFuture<Void> closeAsync();
```

Comment:
:wrench: I would prefer to use the interface `ActorFuture` instead of the concrete implementation here.